### DISCUSSION TITLE
Forward traffic

### DISCUSSION THREAD

**Discussion by AnnaPetrovaWer** [XTLS/Xray-core#3013] [Q&A]
Good day! 
I wanted to consult with you regarding implementing certain functionalities using xray-core.

The situation involves:
1) A foreign server
2) The use of vless+tcp+reality+flow
3) A local server within the country

The goal is to direct traffic from the client to the foreign server and, based on specific geographical conditions, either send it directly or redirect it to a local server in my country using WG/SS if it meets certain rules.

RULES:

```json
{
    "outboundTag": "local_route",
    "ip": [
        "geoip:fi",
    ],
    "type": "field"
}
``` 

OUTBOUND:

```json
{
      "tag": "local_route",
      "protocol": "wireguard",
      "settings": {…}
}
``` 

I've managed a solution but feel it might be rudimentary and am seeking advice from esteemed colleagues for a more appropriate and optimized approach.

Thank you!

**Comment by qwerr0** [comment]
You can write multiple rules to match GEOIP of nearby countries to proxy servers.
{ "ip": ["geoip:us", "geoip:ca", ...], "outboundTag": "my_us_proxy_server" },
{ "ip": ["geoip:de", "geoip:fr", "geoip:"gb", ...], "outboundTag": "my_eu_proxy_server" }
However, relying solely on IP matching is inaccurate, and obtaining a corresponding routing table for each IP is also impractical.

**Reply by andr79hbr** [reply]
Can you please write an example of how to configure routing, outbound and inbound settings for the first and second server?

I tried to write it in the documentation, but all my attempts failed.
